small/lsregion_asan.c

```diff
@@ -30,17 +30,56 @@
  */
 #include "lsregion.h"
 
+SMALL_NO_SANITIZE_ADDRESS static struct lsregion_allocation *
+spare_alloc(struct lsregion *lsregion)
```

Comment:
Let's keep the same order across small code: `static SMALL_NO_SANITIZE_ADDRESS lsregion_allocation *`.

Reply:
Fixed:
```diff
diff --git a/small/lsregion_asan.c b/small/lsregion_asan.c
index 2a8c560..24fece5 100644
--- a/small/lsregion_asan.c
+++ b/small/lsregion_asan.c
@@ -30,7 +30,7 @@
  */
 #include "lsregion.h"
 
-SMALL_NO_SANITIZE_ADDRESS static struct lsregion_allocation *
+static SMALL_NO_SANITIZE_ADDRESS struct lsregion_allocation *
 spare_alloc(struct lsregion *lsregion)
 {
        if (!rlist_empty(&lsregion->allocations)) {
@@ -43,7 +43,7 @@ spare_alloc(struct lsregion *lsregion)
        return NULL;
 }
 
-SMALL_NO_SANITIZE_ADDRESS static struct lsregion_allocation *
+static SMALL_NO_SANITIZE_ADDRESS struct lsregion_allocation *
 alloc_reserve(struct lsregion *lsregion, size_t size, size_t alignment)
 {
        struct lsregion_allocation *alloc = small_asan_alloc(size, alignment,
```

Resolving, since the PR is approved.